report/assets/templates.html

```diff
@@ -611,7 +613,7 @@
   </style>
 
   <a href="#" class="lh-gauge__wrapper lh-gauge--pwa__wrapper">
-    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60" class="lh-gauge lh-gauge--pwa">
+    <svg viewBox="0 0 60 60" class="lh-gauge lh-gauge--pwa">
```

Comment:
This is interesting. I remember working through this to see exactly what was necessary for the PWA badge SVG, but I'm not sure why the `xmlns` would have been necessary for inline svg. Maybe it was just for clarity?

Reply:
this helped me https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed

only xmlns still around now is from the CSS background images data url. those aren't inline HTML and are their own "requests" served as SVG so I figured they need the attribute.